nurseconnect/forms.py

```diff
@@ -0,0 +1,44 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from molo.profiles.forms import EditProfileForm
+from molo.profiles.models import UserProfile
+
+
+class NurseConnectEditProfileForm(EditProfileForm):
```

Comment:
`EditProfileForm` so happens to be a `ModelForm` and by definition, it is not best practise to use it for the purpose you're trying to achieve. The fields `first_name` and `last_name` are not part of the `UserProfile` model.

Rather use a normal form and explicitly list all the fields required there.